backend/src/libs/types/repository.type.ts

```diff
@@ -3,7 +3,7 @@ type Repository<T = unknown> = {
   findAll(): Promise<T[]>;
   create(payload: unknown): Promise<T>;
   update(payload: unknown): Promise<T>;
-  delete(): Promise<boolean>;
+  delete(id: number): Promise<number>;
```

Comment:
Why do we return number here?

Reply:
`  public delete(id: number): Promise<number> {
    return this.chatModel.query().deleteById(id).execute();
  }`
  Because this will return a number, so that's why in service: I have this `return Boolean(await this.chatRepository.delete(id));`